pkg/dbal/Client/DbalDriver.php

```diff
@@ -72,6 +72,7 @@ public function createTransportMessage(Message $message)
         $transportMessage->setMessageId($message->getMessageId());
         $transportMessage->setTimestamp($message->getTimestamp());
         $transportMessage->setDeliveryDelay($message->getDelay());
+        $transportMessage->setTimeToLive($message->getExpire());
```

Comment:
should be `$message->getExpire() * 1000` as clients message [uses seconds](https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/enqueue/Client/Message.php#L43) where transport [uses miliseconds](https://github.com/php-enqueue/enqueue-dev/blob/master/pkg/dbal/DbalMessage.php#L40)